spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/config/LoadBalancerAutoConfiguration.java

```diff
@@ -50,7 +51,8 @@ public LoadBalancerClientFactory loadBalancerClientFactory() {
 	}
 	
 	@Bean
-	public ReactiveLoadBalancerExchangeFilterFunction loadBalancerExchangeFilterFunction() {
+	@ConditionalOnBean(LoadBalancerClientSpecification.class)
```

Comment:
That's incorrect. See https://github.com/spring-cloud/spring-cloud-commons/blob/60d2a1ead3f5a0c4f5347978a60d2206453080e5/spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/reactive/ReactiveLoadBalancerAutoConfiguration.java#L22 for what to do.

Reply:
done